contracts/SCO30_Jonty_VipersHead.Altis/edit_me/briefing/situation.html

```diff
@@ -0,0 +1,19 @@
+<font color="#e3d310">EMPLOYER</font>
+Name: Altian Defence Force Intelligence (ADFI) Official military intelligence,
+desperate to prevent a major terrorist attack, but limited resources and
+political pressure. Contact: Alexios Markou, hardened veteran with a pragmatic
+streak.
```

Comment:
```suggestion
Name: Altian Defence Force Intelligence (ADFI)
Official military intelligence, desperate to prevent a major terrorist attack, but limited resources and political pressure. Contact: Alexios Markou, hardened veteran with a pragmatic streak.
```

Keep the name on its own line, no need to do your own formatting on the longer content